auth.ts

```diff
@@ -1,18 +1,28 @@
 import NextAuth from 'next-auth'
 import GitHub from 'next-auth/providers/github'
 
+declare module 'next-auth' {
```

Comment:
This is moved from `next-auth.d.ts` as TS declaration merging would cause an issue if the declaration file has the same name as the package (`next-auth`)